src/backend/tasks_io/datafeeds/parsers/fms_api/fms_api_event_list_parser.py

```diff
@@ -114,8 +125,8 @@ def parse(self, response: Dict[str, Any]) -> Tuple[List[Event], List[District]]:
                 if code == "week0"
                 else self.EVENT_TYPES.get(api_event_type, None)
             )
-            if api_event_type == "championshipdivision" and self.season != 2022:
-                # 2022 only has one championship and the API uses ChampionshipSubdivision
+            if api_event_type == "championshipdivision" and self.season < 2022:
+                # 2022 onward has one championship and the API uses ChampionshipSubdivision
```

Comment:
I realize now reading this back again how ominous and prophetical it sounds... but 🤷